### PR TITLE
Create task03.py

### PR DESCRIPTION
**Pull request by MACE4GITHUB**
Learning objectives are done:
- Collections: list, dict, tuple
- Work with str
- Iteration techniques: for loop, enumerate(), dict.iteritems(), 
- Tuple unpacking, aka el1, el2 = my_tuple2
- in operator, e.g. some_value in my_list

### CODE REVIEW

File: tasks/task03/task03.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                result[word] = {"count": 1, "first_line": index}
+    return result.items()
+
+def print_line(info):
+    print('%-12s%-12s%-12s' % (info[0], info[1], info[2]))
+
+def print_statistics(items):
+    print_line(["word", "count", "first line"])
+    for item in items:
+        print_line([item[0], item[1]["count"], item[1]["first_line"]])
```
Comment (hhalka) [inline tasks/task03/task03.py]: I think that for more easier understanding you could  add a word itself into the `result[word]` dict.
in this case you can return the whole `result` from `create_statistics`, loop over `items.values()` and use `print_line([item['word'], item["count"], item["first_line"]])`.
still it's just an approach to consider

File: tasks/task03/task03.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        words = _string.lower().translate(str.maketrans(dict.fromkeys(string.punctuation))).split()
+        for word in words:            
+            if word in result.keys():
+                result[word]["count"] += 1
+            else:
+                result[word] = {"count": 1, "first_line": index}
+    return result.items()
+
+def print_line(info):
+    print('%-12s%-12s%-12s' % (info[0], info[1], info[2]))
```
Comment (hhalka) [inline tasks/task03/task03.py]: I would say there is no point to add a list with entries as a parameter, you can just pass all parameters one by one

**Review by hhalka — APPROVED** [review]
nice PR,
consider comments more like possible ways to do the same thing

File: tasks/task03/task03.py (hunk 1 above)
Comment (MACE4GITHUB) [reply tasks/task03/task03.py]: ok

File: tasks/task03/task03.py (hunk 2 above)
Comment (MACE4GITHUB) [reply tasks/task03/task03.py]: ok

**Pull request merged**